remote_config/src/desktop/remote_config_desktop.cc

```diff
@@ -582,11 +582,11 @@ const ConfigInfo RemoteConfigInternal::GetInfo() const {
   return configs_.metadata.info();
 }
 
-ConfigUpdateListenerRegistration*
+ConfigUpdateListenerRegistration
 RemoteConfigInternal::AddOnConfigUpdateListener(
     std::function<void(ConfigUpdate&&, RemoteConfigError)>
         config_update_listener) {
```

Comment:
is parameter config_update_listener used inside this function?

Reply:
Technically no, but that is just because the desktop implementation is a stub for now. The Android and iOS implementations make use of the listener, and desktop will once it is implemented.